### PR TITLE
Addition of barking vulps

### PR DESCRIPTION
**Pull request by KillerTheFlareon**
## Short description

Giving the vulps the bark emote. That's about it
## Why we need to add this

In "historical" portrayals of vulps they all have bark emotes, and many players that come from delta-v or other servers with vulps will often try to "bark!" without success as we lack the emote here. This intends to rectify that. 
## Media (Video/Screenshots)

https://github.com/user-attachments/assets/05320a9f-a3e8-415f-800d-5af1ebb2ddd6

## Checks

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/_StarLight/Body/Implants/eye.yml
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Body/Implants/eye.yml:1]: Why????

File: Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/eye_implant_omni.png
Comment (Rinary1) [inline Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/eye_implant_omni.png:1]: ????

File: Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/meta.json
Comment (Rinary1) [inline Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/meta.json:1]: ?????

**Review by Rinary1 — CHANGES_REQUESTED** [review]

**Comment by fqqf** [comment]
you forked wrong repository, and now having commits from other person added to this PR. to fix that, remove these commits in your local git repository, and then do ```git push --force```, it will update PR
image

File: Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/meta.json
Comment (KillerTheFlareon) [reply Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/meta.json:1]: Forked wrong github, fixed.

File: Resources/Prototypes/_StarLight/Body/Implants/eye.yml
Comment (KillerTheFlareon) [reply Resources/Prototypes/_StarLight/Body/Implants/eye.yml:1]: Forked wrong github, fixed.

File: Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/eye_implant_omni.png
Comment (KillerTheFlareon) [reply Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/eye_implant_omni.png:1]: Forked wrong github, fixed.

**Comment by KillerTheFlareon** [comment]
I believe I undid all JustAdler's commits. Please check to see if the changes saved.

File: Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/meta.json [hunk 1]
```diff
@@ -24,6 +24,6 @@
     },
     {
       "name": "eye_implant_syndicate"
-    }
+    },
```
Comment (fqqf) [inline Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/meta.json]: comma... 😭

Comment (KillerTheFlareon) [reply Resources/Textures/_Starlight/Objects/Specific/Medical/implants.rsi/meta.json]: GUH. Fixed

File: Resources/Locale/en-US/_Starlight/chat/emotes.ftl [hunk 2]
```diff
@@ -4,6 +4,8 @@ chat-emote-name-hisses = hisses
 # Name
 chat-emote-name-howl = Howl
 chat-emote-name-growl = Growl
+chat-emote-name-bark = Bark
 # Msg
 chat-emote-msg-howl = howls
-chat-emote-msg-growl = growls
\ No newline at end of file
+chat-emote-msg-growl = growls
+chat-emote-msg-bark1 = barks
```
Comment (fqqf) [inline Resources/Locale/en-US/_Starlight/chat/emotes.ftl]: why bark1 though?

**Comment by fqqf** [comment]
and it also looks like rinary reverted changes himself lol #135

**Comment by JustAdler** [comment]
yeah i think it's better to work on https://github.com/ss14Starlight/space-station-14/pull/135 so that you don't miss anything here

**Comment by KillerTheFlareon** [comment]
damn, wish i knew that before i fixed everything here lol

**Pull request closed without merging**